pkg/osquery/tables/execparsers/repcli/parser.go

```diff
@@ -0,0 +1,171 @@
+package repcli
+
+// repcli is responsible for parsing the output of the CarbonBlack
+// repcli sensor status utility. Some of the output format has
+// changed from the published documentation, as noted here:
+// https://community.carbonblack.com/t5/Knowledge-Base/Endpoint-Standard-How-to-Verify-Sensor-Status-With-RepCLI/ta-p/62524
+
+import (
+	"bufio"
+	"io"
+	"strings"
+	"unicode"
+)
+
+func NewStatusTemplate() map[string]map[string]any {
```

Comment:
Do you need to export this?

But aside from that, I'm leery of needing to pre-define known fields. It tends change out from under us

Reply:
I think I led @zackattack01 astray by suggesting that we should hard code the data we want.

I was originally worried that counting spaces would be unreliable, but upon reflection and discussion, I think the chance of changing the data keys is higher than the chance of changing the spaces.

Also, we could dynamically determine the length of an indent. Just save the len to a var the first time we encounter a nested header and use that to determine nesting level.

Reply:
Yes, I think you don't have to assume the _length_ of whitespace, just that there is whitespace and it increases when you go in a level. (Python style, I guess?)

I would put the likelihood of data changes much higher than the change at reformatting. And if it reformats, we'll need a new parser anyhow

Reply:
Going to resolve this (no longer relevant), thank you both for the feedback